pkg/controller/bastion/firewallrules.go

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bastion
+
+import (
+	"strconv"
+
+	"google.golang.org/api/compute/v1"
+)
+
+//IngressAllowSSH ingress rule to allow ssh access
```

Comment:
```suggestion
// IngressAllowSSH ingress rule to allow ssh access
```

Reply:
committed in https://github.com/gardener/gardener-extension-provider-gcp/pull/288/commits/6ee5df4bf19f075d65c030c90aca558b5335ed07